### ISSUE TITLE
Application is not rendered after i give the base url http://localhost:<port>/

### ISSUE THREAD

**Issue by inspiremithun** [romulets/spring-rest-example#1]
Hi I have imported the source from GitHub. when I try to run application .In console it shows everything is fine . but when I try to access the application using the base url. noting I rendered in browser. kindly help me

**Comment by romulets** [comment]
Are you sure no errors are shown in your console? What server are you using?

**Comment by inspiremithun** [comment]
I am using embedded tomcat server

On 02-Nov-2017 9:50 AM, "Rômulo M. de Farias" <notifications@github.com>
wrote:

Are you sure no errors are shown in your console? What server are you using?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/romulets/SpringRestExample/issues/1#issuecomment-341313900>,
or mute the thread

**Comment by romulets** [comment]
No js console errors right? And in the Tomcat catalina logs, no error too?

What base url are you using?

**Comment by inspiremithun** [comment]
Yes I didn't get any errors. I will send you the screen shot of my server
cosole

**Comment by inspiremithun** [comment]
[consolelog.txt](https://github.com/romulets/SpringRestExample/files/1438957/consolelog.txt)

I have attached  my server console for your reference. Base url is "http://localhost:8181/ "  . when I use this url application didn't loaded . kindly help me to resolve this issue

When I try to access this url I got 404 not found  in browser

**Comment by romulets** [comment]
Have you tried http://localhost:8181/rest-example ?

**Comment by inspiremithun** [comment]
Yes I tried that too but didn't came

**Comment by romulets** [comment]
What are your dependencies? Could you please fork the project and up all the update that you've made? This way I can see what you've done so far

**Comment by inspiremithun** [comment]
Ok sure Thanks

**Comment by romulets** [comment]
Have you solved it? @inspiremithun

**Comment by inspiremithun** [comment]
No I can't able to find out the reason for the issue
